app/components/ui/button.tsx

```diff
@@ -67,3 +54,18 @@ export function Button({ variant, buttonStyle, icon, disabled, children, ...prop
     </button>
   );
 }
+
+type IconButtonProps = ButtonHTMLAttributes<HTMLButtonElement> & {
+  'aria-label': string;
+  children: React.ReactNode;
+};
+
+export function IconButton({ 'aria-label': ariaLabel, children, ...props }: IconButtonProps) {
```

Comment:
NIT: It's possible to avoid stating `'aria-label': ariaLabel`, since the spread `props` already will add this for us.

Reply:
Good catch! The rest parameter called ...props will collect all the props for the button element.